agent_code/auto_bomber/model.py

```diff
@@ -32,19 +34,27 @@ def select_best_action(self, game_state: dict, agent_self):
         choice = np.random.choice(top_3_actions, p=[0.9, 0.05, 0.05])
         return config.ACTIONS[choice]
 
-    def fit_model_with_transition_batch(self, transitions: Transitions):
+    def fit_model_with_transition_batch(self, transitions: Transitions, round: int):
+        loss = []
```

Comment:
[Optional] The size is known here, thus we can create a fixed size numpy array or?

Reply:
I don't think the size is known previously, as it depends on whether x_all_t.size is zero, right?

Reply:
yep you are correct, oversaw the if statement